runtime-attach/src/main/java/co/elastic/otel/agent/attach/RuntimeAttach.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package co.elastic.otel.agent.attach;
+
+import io.opentelemetry.contrib.attach.core.CoreRuntimeAttach;
+
+public class RuntimeAttach {
+
+  public static void attachJavaagentToCurrentJvm() {
+    CoreRuntimeAttach distroRuntimeAttach = new CoreRuntimeAttach("/otel-agent.jar");
```

Comment:
Can we use any name for the embedded jar? If yes, I'd prefer `edot-agent.jar`

Reply:
Yes, we definitely can, I'll change it to `edot-agent.jar` then, the name here comes from copy-pasting in the upstream implementation, but there is no "convention" to make it automatically included.